static/i18n/fr-FR.json

```
@@ -0,0 +1,547 @@
+  "sort.alphabetical": "A à Z",
+  "sort.reverse": "Z à A",
+  "layout.list": "Vue liste",
+  "layout.small": "Vue petite",
+  "layout.detailed": "Vue détaillée",
+  "layout.pageSize": "Taille de page",
+  "pagination.first": "Première page",
+  "pagination.previous": "Page précédente",
+  "pagination.next": "Page suivante",
+  "pagination.last": "Dernière page",
+  "device.by": "par",
+  "device.noPricing": "Aucun prix disponible",
```

Biome (1.9.4)

[error] The key device.by was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)